src/com/intellij/advancedExpressionFolding/NumberLiteral.java

```diff
@@ -10,8 +10,6 @@
 
 import java.util.ArrayList;
 
-import static sun.misc.FloatingDecimal.toJavaFormatString;
```

Comment:
Internal API, not available in Java 11. Float.toString and Double.toString are public alternatives.